lib/Cake/Network/CakeResponse.php

```
@@ -1336,7 +1336,7 @@ public function file($path, $options = array()) {
 			'download' => null
 		);
 
-		if (strpos($path, '..') !== false) {
+		if (strpos($path, '../') !== false || strpos($path, '..\\') !== false) {
```

I would rather throw the exception here ;) But that is not BC. So meeh.